app/src/main/res/drawable/audio_seekbar_background.xml

```diff
@@ -1,8 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape xmlns:android="http://schemas.android.com/apk/res/android"
-  android:shape="line">
-  <solid android:color="@android:color/transparent" />
-  <stroke
-    android:width="4dp"
-    android:color="@color/white" />
-</shape>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:id="@android:id/background">
+    <shape>
+      <solid
+        android:color="@color/white"/>
+
+      <corners android:radius="7dp" />
+    </shape>
+  </item>
+  <item android:id="@android:id/progress">
+    <clip>
+      <shape>
+        <solid
+          android:color="@color/colorAccent"/>
+        <corners android:radius="7dp" />
```

Comment:
Shouldn’t all dimensions be in multiples of 4dp per material guidelines?

Reply:
Done